packages/js/src/generators/library/library.ts

```diff
@@ -580,5 +578,18 @@ function ensureBabelRootConfigExists(tree: Tree) {
   });
 }
 
+function getOutputPath(options: NormalizedSchema, destinationDir?: string) {
+  const parts = ['dist'];
```

Comment:
small nitpick but I am not sure what `parts` means

Reply:
path parts / segments - I think it should be fine between the name containing path + sm function + ends with `joinPathFragments`